pkg/scheduler/internal/queue/scheduling_queue.go

```diff
@@ -900,7 +905,7 @@ func (p *PriorityQueue) newQueuedPodInfo(pod *v1.Pod, plugins ...string) *framew
 	return &framework.QueuedPodInfo{
 		PodInfo:                 podInfo,
 		Timestamp:               now,
-		InitialAttemptTimestamp: now,
+		InitialAttemptTimestamp: &now,
```

Comment:
Given we want to exclude the duration spent in PreEnqueue, I'd expect it to be set as nil here. And then initialized in `addToActiveQ` when the pod is NOT gated for the first time.